CONTRIBUTING.md

```diff
@@ -21,6 +21,12 @@ consider this the window for comments.
 * All tests must pass on Travis CI for the merge to occur.
 * All changes must not introduce superfluous changes or whitespace errors.
 * All changes should adhere to the coding standard.
+* All commits should adhere to the git commit message guidlines described
+here: https://chris.beams.io/posts/git-commit/ with the following exceptions.
+ * We allow commit subject lines less than 80 characters.
```

Comment:
Did you mean s/less/more ? Otherwise I'm not sure I follow this sentence.

Reply:
haha yeah whoops, long day